Userland/Utilities/sed.cpp

```diff
@@ -833,6 +833,13 @@ static ErrorOr<CycleDecision> apply(Command const& command, StringBuilder& patte
         break;
     case '#':
         break;
+    case 'w': {
+        auto const& w_args = command.arguments->get<WArguments>();
+        auto output_file = TRY(Core::File::open(w_args.output_filepath, Core::File::OpenMode::Write | Core::File::OpenMode::Append));
+        TRY(output_file->write_until_depleted(TRY(pattern_space.to_byte_buffer())));
+        TRY(output_file->write_until_depleted("\n"sv.bytes()));
```

Comment:
nit: `TRY(output_file->write_value('\n'));` is a bit shorter (but as-is is fine too, up to you)